src/component.vue

```diff
@@ -35,6 +35,14 @@ export default {
       type: String,
       default: () => defaults.thousands
     },
+    max: {
```

Comment:
The component did not work until I changed line 5:
`         v-money="{precision, decimal, thousands, prefix, suffix, min, max}"
`

Reply:
Awesome! Good catch! I made this change on the PR